jobs.md

```diff
@@ -4,7 +4,7 @@ sidebar_link: true
 sidebar_sort_order: 500
 ---
 
-The Boston Python group is focused on building a community in the Boston area among Python people.  We are always interested to hear about jobs relevant to members of the community. Such postings can be sent to our mailing list or posted to the #jobs channel on our Slack workspace or both.
+The Boston Python group is focused on building a community in the Boston area among Python people.  We are always interested to hear about jobs relevant to members of the community. Such postings can be sent to our mailing list or posted to the #jobs channel on our [Slack workspace][slack] or both.
```

Comment:
I hadn't linked mailing list or Slack in this paragraph because I wanted them to read the guideline before rushing off to post.  The links are later in the page.